### ISSUE TITLE
bug(ios): The FirstVoices keyboard version 16.0.144 is not working on the iPad

### ISSUE THREAD

**Issue by bharanidharanj** [keymanapp/keyman#10479]
### Describe the bug

I observed that the FirstVoices keyboard isn't functioning in the iPad emulator.

### Reproduce the bug

1. Install FirstVoices 16.0.144 stable build in iPad.
2. Enable 'Haisla'  keyboard.
3. Enable 'Sencoten' keyboard.
4. Set FirstVoices keyboard as the system keyboard. 
5. Open Safari or Chrome browser.
6. Click on the Search bar. 
7. Click the Globe key on the OSK. 
8. Here, we may notice that the 'FirstVoices' keyboard is available in the keyboard menu list. 
9. Select 'FirstVoices' keyboard option. 

Here, I noticed that the FirstVoices keyboard was not displayed after selecting it. Seems to be an issue. 

Note: I am not able to reproduce this issue in iPhone 13 or iPhone emulator. 

..FirstVoices keyboard is not working 
   
   <img width="300" src="https://github.com/keymanapp/keyman/assets/19683143/5adf4a63-e253-4762-bcf5-4720d79e3849">

..FirstVoices keyboard version 

   <img width="300" src="https://github.com/keymanapp/keyman/assets/19683143/ff16dfb2-f395-4f56-87cf-e1a09f09f1b6">

### Expected behavior

The FirstVoices keyboard should function properly on the iPad.

### Keyman apps

- [ ] Keyman for Android
- [X] Keyman for iPhone and iPad
- [ ] Keyman for Linux
- [ ] Keyman for macOS
- [ ] Keyman for Windows
- [ ] Keyman Developer
- [ ] KeymanWeb
- [ ] Other - give details at bottom of form

### Keyman version

16.0.144 stable

### Operating system

iOS 15.0, iOS 17.0

### Device

iPad Air (5th generation)

**Comment by sgschantz** [comment]
I could not reproduce with iOS 17.2 on an iPad Pro (actual device) running Keyman 16.0.144 stable. Did you have Allow Full Access enabled? That should not make a difference with iOS 15, but it blocks the keyboard from displaying on iOS 16 and later.

@bharanidharanj Does this look like the blank keyboard issue, #8566, or is it distinct from that? Can you provide more details on the exact behavior (or lack of behavior) that you see.

**Comment by mcdurdin** [comment]
Just assigning @sgschantz so we don't lose this in the pre-beta noise. @bharanidharanj when you get a chance can you follow up?

**Comment by bharanidharanj** [comment]
> I could not reproduce with iOS 17.2 on an iPad Pro (actual device) running Keyman 16.0.144 stable. Did you have Allow Full Access enabled? That should not make a difference with iOS 15, but it blocks the keyboard from displaying on iOS 16 and later.
> 
> @bharanidharanj Does this look like the blank keyboard issue, #8566, or is it distinct from that? Can you provide more details on the exact behavior (or lack of behavior) that you see.


> Just assigning @sgschantz so we don't lose this in the pre-beta noise. @bharanidharanj when you get a chance can you follow up?

@mcdurdin Sure. I will do it.

**Comment by bharanidharanj** [comment]
@sgschantz 

> I could not reproduce with iOS 17.2 on an iPad Pro (actual device) running Keyman 16.0.144 stable. Did you have Allow Full Access enabled? That should not make a difference with iOS 15, but it blocks the keyboard from displaying on iOS 16 and later.
> 
> @bharanidharanj Does this look like the blank keyboard issue, #8566, or is it distinct from that? Can you provide more details on the exact behavior (or lack of behavior) that you see.

@sgschantz To reproduce this issue, first, we should install Keyman 16.0.144 from the App store. 2. Set Keyman as the default system keyboard. Close Keyman In-App. 3. Install the FirstVoices keyboard. Enable any two FirstVoices keyboard. 4. Set FV keyboard as the system default keyboard. 5. Open Safari or Chrome browser. 6. Click the Search bar. We may notice that the OSK appears on the screen. 7. Click the globe key. 8. Click the FirstVoices keyboard. Here, the FV OSK does not appear on the screen. Could you please try the above steps? Thanks.

**Comment by sgschantz** [comment]
Is this solely seen in FirstVoices or also in iOS?  I was assuming that this was a Keyman issue because it only says 'ios' in the subject, regardless of what the issue contained. The most recent steps to reproduce, mostly reference Keyman rather than FV.

I cannot reproduce with FV 16.0.144 and iOS 17.2.

Can you reproduce this with 'Allow Full Acess' enabled or with iOS prior to 16?

**Comment by bharanidharanj** [comment]
@sgschantz Yes. I enabled the 'Allow Full Access' option. I can reproduce this issue in iPad 15.5 (9th generation) simulator. Here, is the video file for reference, 

https://github.com/keymanapp/keyman/assets/19683143/ec60d2c4-e495-4bdd-9038-a01c9e1161f9

**Comment by sgschantz** [comment]
Thanks for including the video. It seems the keyboard appears as expected, but the FV keyboard picker menu is not populated with the newly added keyboards.

I was able to reproduce this in the simulator on an 8th gen iPad running iOS 14.4. (I'm getting errors in Xcode when I try to  set up the same simulator as you.)

I am not seeing this on an actual iPad.

It seems like we've discussed something similar in the past. Do you know if this is limited to Keyman 16 on iPad simulators only?

**Comment by bharanidharanj** [comment]
@sgschantz Yes, Shawn. It seems that it is limited to Keyman 16 on iPad simulator. Tried the same test on Keyman 17.0.256 build on iPad simulator and it is working fine.

**Comment by sgschantz** [comment]
moving to 18 as this is not reproducible on a real device but still may indicate some timing issue
